packages/client/hmi-client/src/python/PyodideWorker.ts

```diff
@@ -84,9 +84,16 @@ const parseExpression = (expr: string) => {
 	// function to convert expression to mathml
 	let result = pyodide.runPython(`
 		eq = sympy.S("${expr}", locals=_clash)
-		sympy.mathml(eq)
+		sympy.mathml(eq, printer="presentation")
```

Comment:
Is this a parameters to choose between _presentation_ and _content_ MathML?
If yes, could you add a small comment like: 
```suggestion
		sympy.mathml(eq, printer="presentation") // printer refers to the type of MathML used "presentation" or "content"
```

Reply:
oh it got changed to presentation mathML? That is going to have a problem when we save expressions, so I guess we need the function to return both.

Reply:
addressed this